superchain/Dockerfile

```diff
@@ -1,29 +1,42 @@
 FROM amazonlinux:2
 
+# Install deltarpm as it can speed up the upgrade processes
+RUN yum -y install deltarpm
+
 # Install .NET Core, mono & PowerShell
 ENV DOTNET_CLI_TELEMETRY_OPTOUT=true                                                                                    \
-    DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true
+    DOTNET_RUNNING_IN_CONTAINER=true                                                                                    \
+    DOTNET_USE_POLLING_FILE_WATCHER=true                                                                                \
```

Comment:
Same here - is this a development container? It it is only meant to build in a pipeline, all the files are already there - you don't need to ping for source file updates.

Reply:
Like for `FIRST_EXPERIENCE`, Microsoft does it in their container and notes that the `POLLING_FILE_WATCHER` is the only option that is supported in Docker. I'm not certain which context file watchers are being used, so I figured I'd just use the same environment that they do :)

Reply:
Yeah, I noticed where you probably got these from, and checked out the dockerfile. The difference is that that image is designed for development as well, why this may just be used for automation.

The performance impact is probably negligible - but just wanted to give some weight to what these options did, rather than simply adding/removing them.